.vib/discourse/cypress/cypress/support/index.js

```diff
@@ -18,3 +18,11 @@ import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // Discourse header returns an internal resizeObserver
+  // max loop error (from javascript), but the application
+  // works just fine. Returning false here prevents Cypress
+  // from failing the test.
+  return false
```

Comment:
Given it is a specific issue, could you adapt this so the return only affects the known error?

```suggestion
  if (err.message.includes('replace with error message')) {
    return false;
  }
  // we still want to ensure there are no other unexpected
  // errors, so we let them fail the test
```